src/components/Modals/AddContactModal.jsx

```diff
@@ -91,8 +92,8 @@ const AddContactModal = ({ addContact, showAddContactModal, setShowAddContactMod
       aria-labelledby="dialog-title"
       onClose={() => setShowAddContactModal(false)}
     >
-      <FormSection title="Add Contact">
-        <form onSubmit={handleAddContact} autoComplete="off">
+      <FormSection title="Add Contact" id="add-contact-form">
```

Comment:
With the update to using `headingId` for the prop for `<FormSection>` in place of `id` which we'll reserve for elements within the form component, we can update this to `headingId="add-contact-form"`

Reply:
gah, brain off right now, thank you